solenoid/elements/tests/test_elements.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+from requests.exceptions import HTTPError
+
+from solenoid.elements.elements import (get_from_elements, get_paged,
+                                        patch_elements_record)
+from solenoid.elements.errors import RetryError
+
+
+def test_get_from_elements_success(mock_elements):
+    response = get_from_elements('mock://api.com')
+    assert response == 'Success'
+
+
+def test_get_from_elements_retries_and_raises_exception(mock_elements):
```

Comment:
You might consider using [fixture parametrization](https://docs.pytest.org/en/latest/fixture.html#parametrizing-fixtures) here. This could be rewritten:
```python
@pytest.fixture(params=['409', '500', '504'])
def error(request):
    return f'mock://api.com/{request.param}'

def test_get_from_elements_retries_and_raises_exception(mock_elements, error):
    with pytest.raises(RetryError):
        get_from_elements(error)
    assert mock_elements.call_count == 3
```

And then a similar thing down below.